src/Symfony/Bundle/SecurityBundle/Resources/config/security_listeners.php

```diff
@@ -32,19 +31,21 @@
 return static function (ContainerConfigurator $container) {
     $container->services()
 
+        ->set('security.authentication.basic_entry_point', BasicAuthenticationEntryPoint::class)
```

Comment:
This service needs to be deprecated as well. 
All deprecated services should be mentioned in the SecurityBundle changelog also